pkg/trustedresources/verifier/verifier.go

```diff
@@ -26,6 +26,10 @@ import (
 
 	"github.com/sigstore/sigstore/pkg/cryptoutils"
 	"github.com/sigstore/sigstore/pkg/signature"
+	_ "github.com/sigstore/sigstore/pkg/signature/kms/aws"        // pull in aws kms libraries
+	_ "github.com/sigstore/sigstore/pkg/signature/kms/azure"      // pull in azure kms libraries
+	_ "github.com/sigstore/sigstore/pkg/signature/kms/gcp"        // pull in gcp kms libraries
+	_ "github.com/sigstore/sigstore/pkg/signature/kms/hashivault" // pull in hashivault kms libraries
```

Comment:
are these blank identifiers here for now because the packages are not yet used? if so, why not import the packages in the PR where they are actually used?

Reply:
@jerop they are imported here for the [`init`](https://github.com/sigstore/sigstore/blob/main/pkg/signature/kms/gcp/client.go#L44) of those packages to be executed, to "register" themselves on some other "package" that is/might be used. 
For example, this is the same technique used by the k8s credential package helper.

Reply:
(I am usually not a fan of that approach, but oh well, sometimes there is no other way around 🙃 )

Reply:
Thank you so much!! @vdemeester. Sorry I was ooo before and not able to answer this comment.

Reply:
The comments here aren't very informative, consider removing them or including a one-line comment that we need to execute these packages' init functions for xyz reasons.